agents/postgres/pgstatmonitor/stat_monitor_cache.go

```diff
@@ -0,0 +1,237 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgstatmonitor
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/percona/pmm-agent/utils/truncate"
+
+	"github.com/AlekSi/pointer"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/go-mysql/query"
```

Comment:
How we missed that?

Reply:
It is called from fingerprint. Postgres package doesnt had fingerprint method.

https://github.com/percona/pmm-agent/pull/183/files#diff-c7bb32cfdcfd2dcb1693bf29def551dcR132

Or do you mean something else?

Reply:
we use mysql package in postgresql. It should not work this way.

Reply:
It was discussed with @askomorokhov, because he help me with fingerprints. So please @BupycHuk, what you suggest how to do that? Without this package.

Reply:
Use https://github.com/lfittl/pg_query_go to normalize query. PTAL https://github.com/lfittl/pg_query_go/blob/master/normalize_test.go#L23

Reply:
🤦  Yeah, it was a suggestion to it temporary - just to run.
Definitely we need to use https://github.com/lfittl/pg_query_go here for prod.

Reply:
@askomorokhov Sorry I missed that it is temporarily.